s/features/chat/models/room/make-chat-room.ts

```diff
@@ -45,7 +45,7 @@ export function makeChatRoom({label, serverRemote, state}: {
 			serverRemote.clear(label)
 		},
 		mute(userId: string) {
-			serverRemote.mute([userId])
+			state.readable.cache.mutedUserIds.includes(userId) === false ? serverRemote.mute([userId]) : null;
```

Comment:
- i think we should remove this change -- since the button will be disabled, the user cannot trigger this `mute` method anyways
- instead, we should make a similar change to the serverside logic